src/core/plugin/luapi_application.h

```diff
@@ -883,7 +905,11 @@ static int applib_getStrokes(lua_State* L) {
     std::vector<Element*> elements = {};
     Control* control = plugin->getControl();
 
-    if (type == "layer") {
+    // Discard any extra arguments passed in
+    lua_settop(L, 1);
+    luaL_checktype(L, 1, LUA_TSTRING);
+
+    if (type == "Layer") {
```

Comment:
I introduced this spelling mistake again at this point (probably branched off before #4832 got merged).

Reply:
Any way to cleanly fix this? (merge the other PR a second time is not possible I think, and changing #4837 isn't possible anymore as well, is it?)

Reply:
The cleanest way would be to fix this in a separate PR that I can merge right away.